wfcGen.py

```diff
@@ -0,0 +1,264 @@
+import random
+
+from PIL import Image
+
+
+res = 3
+cropShift = False
+cells = {}
+cropped = []
+for ins in range(4):
+    field = Image.open("image.png")
+    if cropShift:
+
+        field = field.rotate(90*ins)
+
+        SX = field.size[0]
+        SY = field.size[1]

```

Comment:
This can be expressed as
```suggestion
        SX, SY = field.size
```